common/remote_object_test.go

```diff
@@ -256,30 +256,41 @@ func TestMultierror(t *testing.T) {
 		initial error
 		errs    []error
 		exp     []error
+		expStr  string
 	}{
 		{
 			name:    "initial error is nil",
 			initial: nil,
 			errs:    []error{mockErr1},
 			exp:     []error{mockErr1},
+			expStr:  mockErr1.Error(),
 		},
 		{
 			name:    "initial error is std error",
 			initial: mockErr1,
 			errs:    []error{mockErr2, mockErr3},
 			exp:     []error{mockErr1, mockErr2, mockErr3},
+			expStr: fmt.Sprintf(
+				"3 errors occurred:\n\t* %s\n\t* %s\n\t* %s\n",
+				mockErr1.Error(), mockErr2.Error(), mockErr3.Error()),
 		},
 		{
 			name:    "initial error is multiError",
 			initial: mockMultiErr,
 			errs:    []error{mockErr3},
 			exp:     []error{mockErr1, mockErr3},
+			expStr: fmt.Sprintf(
+				"2 errors occurred:\n\t* %s\n\t* %s\n",
+				mockErr1.Error(), mockErr3.Error()),
 		},
 		{
 			name:    "initial error is wrapped multiError",
 			initial: mockWrappedMultiErr,
 			errs:    []error{mockErr3, mockErr2},
 			exp:     []error{mockWrappedMultiErr, mockErr3, mockErr2},
+			expStr: fmt.Sprintf(
+				"3 errors occurred:\n\t* %s\n\t* %s\n\t* %s\n",
+				mockWrappedMultiErr.Error(), mockErr3.Error(), mockErr2.Error()),
```

Comment:
Here too:

```suggestion
			expStr: fmt.Sprintf(
				"3 errors occurred:\n\t* %s\n\t* %s\n\t* %s\n",
				mockWrappedMultiErr, mockErr3, mockErr2,
			),
```

Reply:
Fixed in [805bf89](https://github.com/grafana/xk6-browser/pull/734/commits/805bf899fd3e5cc0dac0221731416bf375273812).